deckhouse-controller/pkg/controller/module-controllers/release/controller.go

```diff
@@ -110,7 +111,7 @@ func NewModuleReleaseController(
 	preflightCountDown *sync.WaitGroup,
 	logger *log.Logger,
 ) error {
-	lg := logger.With("component", "ModuleReleaseController")
+	lg := log.Default().Named("ModuleReleaseController")
```

Comment:
```suggestion
	lg := logger
```
we already named it when we call NewModuleReleaseController